README.md

```diff
@@ -91,45 +91,64 @@ houses the portion of the workflow that does estimation
 of the DFE.
 
 
-------------------
-
-A small example to simulate genomes with selection is described below.
-
-
-# Simulating selection with stdpopsim
-
-For each HomSap chromosome one can simulate selection in specific regions (e.g. coding region)
-by using the function in stdpopsim and passed to
-`stdpopsim.species.get_annotations` (see [the catalog](https://popsim-consortium.github.io/stdpopsim-docs/latest/tutorial.html#simulating-selection-on-exons) for details). Below is a complete example
-that uses a given annotation to simulate selection on chr20.
-
-```python
-import stdpopsim
+## Running the workflow on a cluster
+Currently we have provided two example `Snakemake` profiles
+that allow a user to run the `analysis2` workflow on quite
+easily. These can be found in `workflows/config/snakemake/oregon_profile`
+and `workflows/config/snakemake/arizona_profile`.
+
+### Cluster profile files
+Each of those directories contains a single file, `config.yaml`,
+that lays out the cluster specific settings needed to launch jobs.
+
+For instance the `oregon_profile/config.yaml`, which is meant
+to run using a cluster with a `slurm` scheduler looks like this
+
+```yaml
+cluster:
+        mkdir -p logs/{rule} &&
+        sbatch
+                --partition=kern,kerngpu
+                --account=kernlab
+                --cpus-per-task={threads}
+                --mem={resources.mem_mb}
+                --time={resources.time}
+                --job-name=smk-{rule}-{wildcards}
+                --output=logs/{rule}/{rule}-{wildcards}-%j.out
+default-resources:
+        - time=60
+        - mem_mb=5000
+        - threads=1
+restart-times: 3
+max-jobs-per-second: 10
+max-status-checks-per-second: 1
+local-cores: 1
+latency-wait: 60
+jobs: 500
+keep-going: True
+rerun-incomplete: True
+printshellcmds: True
+scheduler: greedy
+use-conda: True
+```
 
-species = stdpopsim.get_species("HomSap")
+to adopt this to a new `slurm` cluster a user would have to:
+- change the `partition` value to appropriately named partitions
+- change the `account` name
 
```

Comment:
For other cluster systems, users can find available profiles in https://github.com/Snakemake-Profiles/doc

Reply:
ah nice-- yes i mention this at the end of the README